### DISCUSSION TITLE
Bevy crashes with DefaultPlugins

### DISCUSSION THREAD

**Discussion by njbirth** [bevyengine/bevy#2991] [Q&A] [answered]
I just started trying to become acquainted with Bevy. However, as soon as I add the DefaultPlugins, the program crashes. This is my code:

```Rust
use bevy::prelude::*;

fn main() {
    App::build()
        .add_plugins(DefaultPlugins)
        .run();
}
```

And this is the output:
```
thread 'main' panicked at 'build_output_stream failed with all supported formats: BackendSpecific { err: BackendSpecificError { description: "ALSA function 'snd_pcm_sw_params' failed with error 'EINVAL: Invalid argument'" } }'
```

I don't really know what to do with this error. A web search also didn't produce any meaningful results. All dependencies listed [here](https://github.com/bevyengine/bevy/blob/main/docs/linux_dependencies.md#arch--manjaro) are installed and I'm using the newest versions of Rust and Bevy. It would be great if someone here would have an idea how to solve this problem. If you need more information, just ask and I will provide them if I can.

**Comment by bjorn3** [comment, marked as answer]
Are you using bevy 0.5. I believe this has been fixed by https://github.com/bevyengine/bevy/pull/2269 on the main branch.

**Reply by njbirth** [reply]
Thanks for the quick response. Looks like this is indeed the case. I also found a solution for my problem in the [linked issue](https://github.com/bevyengine/bevy/issues/850#issuecomment-755772936). After I installed `pulseaudio-alsa`everything works now.
